### DISCUSSION TITLE
How to setup Routing Only Flow

### DISCUSSION THREAD

**Discussion by Park-Hyung-Joo** [siliconcompiler/siliconcompiler#2718] [Help (Q&A)]
Hello all, I am trying to build an IP with full-custom cells only. I have my own placement so all I need from the siliconcompiler flow is routing with constraint. Could I get the flow or hint on how to setup such flow? I have seen [this](https://github.com/siliconcompiler/siliconcompiler/discussions/1124) but I guess the notation and code line references were a bit outdated.

**Comment by gadfort** [comment]
@Park-Hyung-Joo it seems like you would want to create a new flow with a single node
```
.... route_flow.py

def setup(chip):
    flow = Flow(chip, "route_flow")
    flow.node("route_flow", "route", route)
    return flow
```

it's possible the openroad tool driver would need to corrected to handle getting a def input, if that is the case let me know and I can add that.
